models.py

```diff
@@ -1193,6 +1193,30 @@ def __init__(self, *args, **keyargs):
        self.fields[field].widget.attrs['required'] = 'required'
 
 
+class UserPref(DictModel):
+  """Describes a user's application preferences."""
+
+  email = db.EmailProperty(required=True)
+  notify_as_starrer = db.BooleanProperty(default=True)
```

Comment:
I can guess, based on the field name what behavior notify_as_starrer implies.  However, it may be worth explicitly adding a comment to remove any potential for ambiguity.